docs/testing.md

```diff
@@ -0,0 +1,55 @@
+# Testing Storefront
+
+This document lists the features and flows supported in Storefront. This is not an exhaustive list.
+
+## WooCommerce core
+- [ ] single product page
+- [ ] shop page / product catalog / product list
+- [ ] search for products
+- [ ] sale products
+- [ ] add to cart, view cart, manage cart contents
+- [ ] use coupon
+- [ ] checkout
+- [ ] log in as customer, view account info
+- [ ] customer can view previous orders
+- [ ] visitor/customer can review products
+- [ ] visitors can view reviews
+- [ ] woo widgets
+
+## WooCommerce blocks
+- [ ] show products
+- [ ] explore / search products
+- [ ] show reviews
+- [ ] all products / grid blocks
+
+## Storefront features
+- [ ] onboarding / welcome notice / create homepage (unset storefront_nux_dismissed)
+- [ ] homepage template
+- [ ] full width page template
+- [ ] sticky add-to-cart header
+- [ ] block editor adapts to full/sidebar width as appropriate (editor.js)
```

Comment:
What does `editor.js` mean in this line?

Reply:
This is an interesting case – I don't fully understand the rationale for the feature, and want to avoid detailing all the specifics of how it works. I added the filename for the code because there might be other aspects to the feature that we need to cover. We might not need this item specifically - it might be covered by `editor styles match front end` for example. 

The goal here is to have a flow item(s) that captures the benefits for merchants. 

A related consideration is how critical/important each flow is. Perhaps we need a column/rating for criticality (e.g. `essential | useful | low priority`.

Reply:
I think it's much clearer now. Thanks for expanding it!